### DISCUSSION TITLE
Multiple job submission

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1685] [Q&A] [answered]
Hi, do you know if it is possible to run more simulations creating a "queue" between them?

**Comment by PaulAltair** [comment]
Hi there, in principal, it is a simple as adding your job submission lines sequentially to a batch script, (write your job submission strings one after the other in a text file with directory changes etc if needed and 'execute' it) this will work in Windows and/or linux

Or in this thread here, for windows mpi running, user @hotaosa has posted a nice gui with queuing capability

https://github.com/orgs/OpenRadioss/discussions/1231#discussioncomment-6380921

**Reply by PaulAltair** [reply, marked as answer]
e.g. this is a script I ran when I was testing something, to submit 5 jobs sequentially in 5 folders (01,02,03,04,05), clear up the rst files, then grep some Mass values from the output and put in a text file:

```
cd 01
/home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/starter_linux64_gf -i CamryOpenRadioss.key -np 4 -nt 4
mpiexec -n 4 /home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/engine_linux64_gf_ompi -i CamryOpenRadioss_0001.rad -nt 1
rm *.rst
cd ../02
/home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/starter_linux64_gf -i CamryOpenRadioss.key -np 4 -nt 4
mpiexec -n 4 /home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/engine_linux64_gf_ompi -i CamryOpenRadioss_0001.rad -nt 1
rm *.rst
cd ../03
/home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/starter_linux64_gf -i CamryOpenRadioss.key -np 4 -nt 4
mpiexec -n 4 /home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/engine_linux64_gf_ompi -i CamryOpenRadioss_0001.rad -nt 1
rm *.rst
cd ../04
/home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/starter_linux64_gf -i CamryOpenRadioss.key -np 4 -nt 4
mpiexec -n 4 /home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/engine_linux64_gf_ompi -i CamryOpenRadioss_0001.rad -nt 1
rm *.rst
cd ../05
/home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/starter_linux64_gf -i CamryOpenRadioss.key -np 4 -nt 4
mpiexec -n 4 /home/psharp/OpenRadioss/OpenRadioss-PaulTesting/exec/engine_linux64_gf_ompi -i CamryOpenRadioss_0001.rad -nt 1
cd ..
tail -n 100000 ./*/CamryOpenRadioss_0000.out | grep -A4 -e "TOTAL MASS" > Masses.txt

````
I save this to a file 'runmyjobs.sh' make executable (`chmod 777 runmyjobs.sh`) then execute, (`./runmyjobs.sh`)
